Foundation/NSTask.swift

```diff
@@ -40,24 +40,24 @@ private var managerThreadRunLoopIsRunningCondition = NSCondition()
 internal let kCFSocketDataCallBack = CFSocketCallBackType.dataCallBack.rawValue
 #endif
 
-private func emptyRunLoopCallback(context : UnsafeMutablePointer<Void>) -> Void {}
+private func emptyRunLoopCallback(context : UnsafeMutablePointer<Void>!) -> Void {}
 
 
 // Retain method for run loop source
-private func runLoopSourceRetain(pointer : UnsafePointer<Void>) -> UnsafePointer<Void> {
+private func runLoopSourceRetain(pointer : UnsafePointer<Void>!) -> UnsafePointer<Void>! {
```

Comment:
This is an unfortunate consequence of the compiler being strict about `@convention(c)` callback types.